assets/airports/kdtw.json

```diff
@@ -0,0 +1,4870 @@
+                ["eny", 1],
+                ["fft", 2],
+                ["gjs", 7],
+                ["jia", 0.5],
+                ["n/fastGA", 1],
+                ["nks", 3.5],
+                ["rpa", 5.5],
+                ["swa", 3.5],
+                ["ual", 0.5]
+            ]
+        },
+        {
+            "origin": "",
+            "destination": "KDTW",
+            "category": "arrival",
+            "route": "BAE.POLAR5.KDTW",
+            "altitude": [11000, 12000],
+            "speed": 320,
+            "method": "random",
+            "rate": 6,
+            "airlines": [
+                ["aal", 4.5],
+                ["asa", 0.5],
+                ["asq", 3],
+                ["awi", 1],
+                ["cpz", 0.5],
+                ["dal", 25.5],
+                ["dal/long", 10],
+                ["edv", 12.5],
+                ["eny", 1],
+                ["fft", 2],
+                ["gjs", 7],
+                ["jbu", 1],
+                ["jia", 0.5],
+                ["n/fastGA", 1],
+                ["nks", 3.5],
+                ["rpa", 5.5],
+                ["skw", 15],
+                ["swa", 3.5],
+                ["ual", 0.5]
+            ]
+        },
+        {
+            "origin": "",
+            "destination": "KDTW",
+            "category": "arrival",
+            "route": "BUF.SPICA2.KDTW",
+            "altitude": [11000, 12000],
+            "speed": 320,
+            "method": "random",
+            "rate": 5,
+            "airlines": [
+                ["aal", 4.5],
+                ["afr/long", 0.5],
+                ["asq", 3],
+                ["awi", 1],
+                ["cpz", 0.5],
+                ["dal", 25.5],
+                ["dal/long", 10],
+                ["dlh/long", 0.5],
+                ["edv", 12.5],
+                ["eny", 1],
+                ["fft", 2],
+                ["gjs", 7],
+                ["jia", 0.5],
+                ["n/fastGA", 1],
+                ["nks", 3.5],
+                ["rja/long", 0.5],
+                ["rpa", 5.5],
+                ["swa", 3.5],
+                ["ual", 0.5]
+            ]
+        },
+        {
+            "origin": "",
+            "destination": "KDTW",
+            "category": "arrival",
+            "route": "HNN.WEEDA2.KDTW",
+            "altitude": 12000,
+            "speed": 320,
+            "method": "random",
+            "rate": 2,
+            "airlines": [
+                ["aal", 4.5],
+                ["asq", 3],
+                ["awi", 1],
+                ["cpz", 0.5],
+                ["dal", 25.5],
+                ["edv", 12.5],
+                ["eny", 1],
+                ["fft", 2],
+                ["gjs", 7],
+                ["jbu", 1],
+                ["jia", 0.5],
+                ["n/fastGA", 1],
+                ["nks", 3.5],
+                ["rpa", 5.5],
+                ["swa", 3.5],
+                ["ual", 0.5]
+            ]
+        }
+    ],
+    "maps": {
```

Comment:
Can we create a map with extended centerlines for all runway configurations, just to keep it flexible? I can help with this when I get back if needed.